go.mod

```diff
@@ -1,15 +1,14 @@
 module github.com/code-ready/crc
 
 require (
-	github.com/BurntSushi/toml v0.3.1 // indirect
 	github.com/DATA-DOG/godog v0.7.13
 	github.com/YourFin/binappend v0.0.0-20181105185800-0add4bf0b9ad
 	github.com/cavaliercoder/grab v2.0.0+incompatible
 	github.com/code-ready/clicumber v0.0.0-20190801094041-02a92c8f7ece
 	github.com/code-ready/goodhosts v0.0.0-20190712111040-f090f3f77c26
 	github.com/code-ready/machine v0.0.0-20190904103148-0fde37b5d95b
 	github.com/docker/go-units v0.4.0
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/golangci/golangci-lint v1.17.1 // indirect
```

Comment:
not sure this one is wanted/needed?

Reply:
this shouldn't be there. `golangci-lint` is not direct dependency for the project like we are not importing it for any file.

Reply:
This happened automatically when modifying running with `make lint` from gitpod.